basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/CoreStep/help.html

```diff
@@ -0,0 +1,9 @@
+<div>
+    <p>
```

Comment:
Possibly OK but interactively check appearance of `<p>` to make sure there is not ugly extra padding.